libraries/AP_Math/quaternion.h

```diff
@@ -21,7 +21,10 @@
 #if MATH_CHECK_INDEXES
 #include <assert.h>
 #endif
-#include <math.h>
+#include <math.h>  // needed for isnan
```

Comment:
Please explain, build system should take care of fixing this problem.

Reply:
I don't know how I did get an error on that, but it seems working now !